fly/rc/target.go

```diff
@@ -166,7 +191,13 @@ func LoadUnauthenticatedTarget(
 		return nil, err
 	}
 
-	httpClient := &http.Client{Transport: transport(insecure, caCertPool)}
+	var clientCertificate []tls.Certificate
+	clientCertificate, err = loadClientCertificate(clientCertPath, clientKeyPath)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
With the CA Cert, we fall back to the existing values if left unspecified. In fact, it looks like there's a test for this: https://github.com/concourse/concourse/blob/9e29396f1a044a585fb2d84d1a7ff70d92212d3e/fly/integration/login_test.go#L284-L285.

Without that fallback, users would need to provide `--client-certificate` and `--client-key` every time they try to refresh their token, which could be a bit tedious. What do you think about adding a fallback here, e.g.

```go
if clientCertPath == "" {
	clientCertPath = targetProps.ClientCertPath
}
...
```

Reply:
On a similar note, maybe it's worthwhile to add a test to `login_test.go` and possibly `sync_test.go`